Makefile

```diff
@@ -29,6 +29,7 @@ endif
 
 REGISTRY ?= $(USER)
 
+.PHONY: build
 build: build-client build-runtime
```

Comment:
Should we remove and/or adjust the `build-runtime` target?

Reply:
Also, should/can we remove the `install $(BINDIR)/$(MIXIN)-runtime$(FILE_EXT) $(PORTER_HOME)/mixins/$(MIXIN)/$(MIXIN)-runtime$(FILE_EXT)` line under the `install` target?

Reply:
We still need the build-runtime (see above).

The install target may still be useful? For when you are doing local development and want to copy it into the porter home. You've been doing that a lot lately, so let me know if this isn't helpful.